Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -229,4 +279,51 @@ lemma free_bot (h : A ≠ ⊥) : A.Free (⊥ : SimpleGraph β) := by
 
 end Free
 
+/-!
+#### Induced containment
+
+A graph `H` *inducingly contains* a graph `G` if there is some graph embedding `G ↪ H`. This amounts
+to `H` having an induced subgraph isomorphic to `G`.
+
+We denote "`G` is contained in `H`" by `G ⊴ H` (`\triangle_left_eq`).
+-/
+
+/-- A simple graph `G` is contained in a simple graph `H` if there exists an induced subgraph of `H`
```

Comment:
```suggestion
/-- A simple graph `G` is inducingly contained in a simple graph `H` if there exists an induced subgraph of `H`
```
This is induced containment not just containment!
(I don't love "inducingly contained" as a name, but I don't have better suggestions)